src/Response.php

```diff
@@ -82,6 +95,46 @@ public function getBody()
         return $this->body;
     }
 
+    /**
+     * @return array
+     */
+    public function getRecords()
+    {
+        return $this->records;
+    }
+
+    /**
+     * @return array
+     */
+    public function getRawResponse()
+    {
+        return $this->response;
+    }
+
+    /**
+     * @return string
+     */
+    public function getScriptResult()
+    {
+        if (isset($this->response['scriptResult']) || array_key_exists('scriptResult', $this->response)){
+            return $this->response['scriptResult'];
+        } else {
+            return;
```

Comment:
This needs to do something else than just "return;". As of PHP 7.1, that is a compile-time error.

https://www.php.net/manual/en/function.return.php